mms/model_service/model_service.py

```diff
@@ -148,22 +139,9 @@ def inference(self, data):
         list of outputs to be sent back to client.
             data to be sent back
         """
-        pre_start_time = time.time()
         data = self._preprocess(data)
-        infer_start_time = time.time()
         data = self._inference(data)
-        post_start_ms = time.time()
         data = self._postprocess(data)
-        post_end_ms = time.time()
-
-        pre_time_in_ms = (infer_start_time - pre_start_time) * 1000
-        infer_time_in_ms = (post_start_ms - infer_start_time) * 1000
-        post_time_in_ms = (post_end_ms - post_start_ms) * 1000
-
-        metrics = self._context.metrics
-        metrics.add_time(PREPROCESS_METRIC, pre_time_in_ms)
```

Comment:
why are the metrics being removed here?

Reply:
1. Those are legacy models
2. They belong to model metrics. We don't have that part implemented yet.